content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,11 +9,24 @@ menu:
     parent: About the project
 ---
 
+## v1.8.5 [2020-04-19]
+
+The InfluxDB Enterprise 1.8.5 release builds on the InfluxDB OSS 1.8.5 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v185-2021-04-19).
+
+### Bug fixes
+
+- Resolve TSM backup "snapshot in progress" error.
+- Make meta queries respect QueryTimeout values.
+- Add labels to verbose output.
```

Comment:
From Ayan, of lamented loss to Influx:

> Prior to this commit, the show shards command will print the shard
> owners value without labels like below:
> 
> {7 ics106019162.ics-eu-1.asml.com:8088} {4 ics106019220.ics-eu-1.asml.com:8088}]
> 
> This is because we use the "%v" verb to print the struct.
> 
> This patch uses the "%+v" verb which instructs Go to include the struct
> member names.
> 
> This will allow for more readable output like below:
> 
> [{ID:7 TCPAddr:ics106019162.ics-eu-1.asml.com:8088} {ID:4 TCPAddr:ics106019220.ics-eu-1.asml.com:8088}]

TLDR from me: 

> This change will affix labels to the values returned in `SHOW SHARDS` output clarifying what is the node ID and what is the TCP address.